src/java.base/share/classes/java/net/Inet6Address.java

```diff
@@ -477,6 +495,128 @@ public static Inet6Address getByAddress(String host, byte[] addr,
         throw new UnknownHostException("addr is of illegal length");
     }
 
+    /**
+     * Creates an {@code InetAddress} based on the provided {@linkplain
+     * Inet6Address##format textual representations} of an IPv6 address.
```

Comment:
"textual representations" -> "textual representation"

Reply:
Fixed in af87fab27148daa982ab1997a4346e8c29be7604